frontend/src/scenes/funnels/funnelLogic.test.ts

```diff
@@ -651,9 +663,12 @@ describe('funnelLogic', () => {
                     },
                 })
         })
-
         it('are not updated when results are loaded, when steps visualisation set, with one funnel step', async () => {
-            // url unset, so syncWithUrl is false
+            initKeaTests()
```

Comment:
Hmmm, I actually wanted to test the former, as mentioned in the comment, where logic is loaded, syncWithUrl is true, *but* you're not on `/insights` and say, are on `experiments`. 

But this works reasonably fine as well, since other places ideally have the prop itself set to false.

Reply:
Ah... sorry... 😅  

Feel free to change it however it should be and I'll be happy to quickly review/merge.